tests/json-fixtures/test_transactions.py

```diff
@@ -102,9 +102,9 @@ def fixture_transaction_class(fixture_data):
         return ByzantiumTransaction
     elif fork_name == ForkName.Constantinople:
         return ConstantinopleTransaction
-    elif fork_name == "ConstantinopleFix":
+    elif fork_name == ForkName.Petersburg:
```

Comment:
oops, I named this as `ConstantinopleFix`

Reply:
I tried to guess without looking and was wrong. :smile: One sec...